src/migrations/grammar.rs

```diff
@@ -27,6 +28,8 @@ pub struct TokenMatch<'a> {
     phantom: PhantomData<&'a u8>,
 }
 
+pub struct TokenStream<'a>(pub Tokenizer<'a>);
```

Comment:
Eventually should use chumsky here too. This doesn't have to be in PR. But I would appreciate a `// TODO` comment here